x-pack/plugin/sql/qa/server/src/main/resources/command.csv-spec

```diff
@@ -9,15 +9,15 @@ showFunctions
 SHOW FUNCTIONS;
 
     name:s       |    type:s
-AVG              |AGGREGATE      
+AVG              |AGGREGATE
```

Comment:
Missed these ones, will revert the whitespace changes in this file.

Reply:
Please do so.

Reply:
Done.